src/components/InputWithTitle/InputWithTitle.tsx

```diff
@@ -0,0 +1,43 @@
+import { H1, Input, Stack, useDeskproAppTheme } from "@deskpro/app-sdk";
+import { UseFormRegisterReturn } from "react-hook-form";
+
+interface Props {
+  title: string;
+  error: boolean;
+  required?: boolean;
+  register: UseFormRegisterReturn;
+  type?: string;
+}
+
+export const InputWithTitle = ({
+  title,
+  error,
+  required,
+  register,
+  type,
+  ...attributes
+}: Props) => {
+  const { theme } = useDeskproAppTheme();
+  return (
+    <Stack vertical style={{ width: "100%", marginTop: "5px" }}>
+      <Stack>
+        <div style={{ color: theme?.colors?.grey80 }}>
+          <H1>{title}</H1>
+        </div>
+        {required && (
+          <Stack style={{ color: "red" }}>
```

Comment:
please don't use hardcoded values, use a theme from the context theme.colors.red100